recipes/cpprestsdk/all/conanfile.py

```diff
@@ -0,0 +1,180 @@
+                  "C++ API design"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Microsoft/cpprestsdk"
+    topics = ("conan", "cpprestsdk", "rest", "client", "http")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "exclude_websockets": [True, False],
+        "exclude_compression": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "exclude_websockets": False,
+        "exclude_compression": False
+    }
+    short_paths = True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == 'Visual Studio':
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires.add("openssl/1.1.1e")
+        if not self.options.exclude_compression:
+            self.requires.add("zlib/1.2.11")
+        if not self.options.exclude_websockets:
+            self.requires.add("websocketpp/0.8.1")
+        self.requires.add("boost/1.72.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        if self.settings.os == "iOS":
+            with open('toolchain.cmake', 'w') as toolchain_cmake:
+                if self.settings.arch == "armv8":
+                    arch = "arm64"
+                    sdk = "iphoneos"
+                elif self.settings.arch == "x86_64":
+                    arch = "x86_64"
+                    sdk = "iphonesimulator"
+                sysroot = tools.XCRun(self.settings).sdk_path
+                toolchain_cmake.write('set(CMAKE_C_COMPILER /usr/bin/clang CACHE STRING "" FORCE)\n')
+                toolchain_cmake.write('set(CMAKE_CXX_COMPILER /usr/bin/clang++ CACHE STRING "" FORCE)\n')
+                toolchain_cmake.write('set(CMAKE_C_COMPILER_WORKS YES)\n')
+                toolchain_cmake.write('set(CMAKE_CXX_COMPILER_WORKS YES)\n')
+                toolchain_cmake.write('set(CMAKE_XCODE_EFFECTIVE_PLATFORMS "-%s" CACHE STRING "" FORCE)\n' % sdk)
+                toolchain_cmake.write('set(CMAKE_OSX_ARCHITECTURES "%s" CACHE STRING "" FORCE)\n' % arch)
+                toolchain_cmake.write('set(CMAKE_OSX_SYSROOT "%s" CACHE STRING "" FORCE)\n' % sysroot)
+            os.environ['CONAN_CMAKE_TOOLCHAIN_FILE'] = os.path.join(os.getcwd(), 'toolchain.cmake')
```

Comment:
See https://github.com/conan-io/conan-center-index/pull/1137
Or bump the cmake mimimum version: https://github.com/conan-io/conan-center-index/pull/1137#issuecomment-601103816

Reply:
@uilianries Just remove these lines?
Users should have their profile properly configured.

Reply:
I think it was an option when we created that recipe, but it's pretty. I agree, it should be removed now as the support is much better